### PR TITLE
Modernize?

### PR DESCRIPTION
**Pull request by movermeyer**
I couldn't get `set-tz` to work in my project. So I've dug into it a bit and did some modernization work at the same time:

* Modernized the version of `ava` used
* Fixed up the failing tests
* Fixed up the `globalSetup` script and instructions.
* Added a CI using GitHub Actions (You need to enable GitHub Actions on the repo before it will run).

### CODE REVIEW

File: tests/set-tz.spec.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+  if (process.platform === 'win32') {
+    t.is(
+      new Date().toString().includes(' GMT+0000 (Greenwich Mean Time)'),
+      true
+    )
+  } else {
+    t.is(
+      new Date().toString().includes(' GMT+0000 (Coordinated Universal Time)'),
+      true
+    )
```
Comment (movermeyer) [inline tests/set-tz.spec.js:16]: Ref: https://github.com/capaj/set-tz/issues/7

**Comment by movermeyer** [comment]
cc @capaj

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      - name: Install Dependencies
+        run: yarn
+      - name: Build Packages
+        run: yarn build
+      - name: Run Tests
+        uses: GabrielBB/xvfb-action@v1.0
```
Comment (capaj) [inline .github/workflows/ci.yml:40]: this GH action should not be needed

File: tests/set-tz-iana.spec.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import test from 'ava'
```
Comment (capaj) [inline tests/set-tz-iana.spec.js:1]: please keep ES6 imports

**Review by capaj — CHANGES_REQUESTED** [review]
thanks for that CI yaml, just revert those imports please and also revert utc.js file

**Comment by movermeyer** [comment]
@capaj, Unfortunately, it's been a year since I opened this PR, so I've forgotten all of the context I had at the time.

I won't be able to bring this PR back into a good state. Sorry. Feel free to use it for inspiration for another PR and/or close this PR.

File: tests/set-tz-iana.spec.js (hunk 3 above)
Comment (movermeyer) [reply tests/set-tz-iana.spec.js:1]: FWIW, `npm test` fails then:

```
➜  set-tz git:(e342209) ✗ npm test

> set-tz@0.2.0 test
> ava tests -s


  Uncaught exception in tests/set-tz-iana.spec.js

  SyntaxError: Cannot use import statement outside a module

  ✖ tests/set-tz-iana.spec.js exited with a non-zero exit code: 1
```

Comment (capaj) [reply tests/set-tz-iana.spec.js:1]: I think ava stopped supporting ES6 syntax out of the box. We need to add a config for it.

**Pull request closed without merging**